plugins/palo_alto_cortex_xdr/plugin.spec.yaml

```diff
@@ -4,13 +4,13 @@ products: [insightconnect]
 name: palo_alto_cortex_xdr
 title: Palo Alto Cortex XDR
 description: Stop modern attacks with the industry's first extended detection and response platform that spans your endpoints, network and cloud data
-version: 3.0.0
-connection_version: 2
+version: 4.1.0
```

Comment:
Version 4.0.0 isn't released yet so good news is we can keep this at 4.0.0 and put the two updates out together.

Reply:
Really good spot Oo